cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -319,6 +322,22 @@ private String rebalanceStateConditionType(KafkaRebalanceStatus status) {
 
         log.info("{}: Rebalance action from state [{}]", reconciliation, currentState);
 
+        if (Annotations.isReconciliationPausedWithAnnotation(kafkaRebalance)) {
+            // we need to do this check again because it was triggered by a watcher
+            KafkaRebalanceStatus status = new KafkaRebalanceStatus();
+
+            Condition pauseCondition = new ConditionBuilder()
+                    .withLastTransitionTime(StatusUtils.iso8601Now())
+                    .withType("ReconciliationPaused")
+                    .withStatus("True")
+                    .build();
```

Comment:
Probably factor out some helper method to build a "paused condition", to save so much code duplication with other places we add this condition.